packages/manager/modules/vps/src/dashboard/migrate/migrate.routing.js

```diff
@@ -9,8 +9,25 @@ export default /* @ngInject */ ($stateProvider) => {
       },
     },
     resolve: {
-      goToMigrateConfirm: /* @ngInject */ ($state) => () =>
-        $state.go('vps.detail.dashboard.migrate.confirm'),
+      migrationTrackingPrefix: /* @ngInject */ (vpsMigration) => {
+        return `vps::vps-migration::from_${vpsMigration.currentPlan}_to_${vpsMigration.newPlan}`;
+      },
+      trackPage: /* @ngInject */ (atInternet, migrationTrackingPrefix) =>
+        atInternet.trackPage({
+          name: migrationTrackingPrefix,
+          type: 'navigation',
+        }),
```

Comment:
Isn't this already handled by `ng-at-internet-ui-router-plugin` ?

Reply:
Need customized page tracking here so i am overriding atInternet, ignoring default tracking.